test/modules/diten/modulescopes.good

```diff
@@ -1,2 +1,2 @@
 modulescopes.chpl:11: In function 'main':
-modulescopes.chpl:13: error: Symbol 'a' undeclared in module 'M3'
+modulescopes.chpl:13: error: cannot find 'a' in module 'M3'
```

Comment:
Making sure I'm understanding this one:  It looks like M3 definitely contains an 'a', so should be easy to find.  Is the point that the `only ;`, as applied to `M1` prevents us from seeing `M2` which prevents us from seeing `M3` or `M3.a`?  If so, it seems like a clearer error message would be one that focused on not being able to find 'M2' rather than 'M3.a' (and maybe rather than saying `cannot find 'M2' in module 'M1'` would say something like `cannot find 'M2' in module 'M1' given current 'use'/'import' statements` or the like, to point the user toward the `use` rather than the module definitions themselves?